App.js

```diff
@@ -37,7 +38,7 @@ export default class App extends React.Component {
   render() {
     return (
       <Provider store={store}>
-        <BottomLinks />
```

Comment:
Do we not need these navigation components anymore?

Reply:
added them back in